toplevel/coqargs.ml

```diff
@@ -501,7 +506,12 @@ let parse_args ~help ~init arglist : t * string list =
     |"-bt" -> Exninfo.record_backtrace true; oval
     |"-color" -> set_color oval (next ())
     |"-config"|"--config" -> set_query oval PrintConfig
+
     |"-debug" -> Coqinit.set_debug (); oval
+    |"-d" | "-D" ->
+      CDebug.set_debug_levels (parse_debug_flags (next ()));
```

Comment:
this should be moved to the injection system after #13418

Reply:
changed